resources/darwin/bin/code.sh

```diff
@@ -3,8 +3,8 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for license information.
 
-function realpath() { python -c "import os,sys; print(os.path.realpath(sys.argv[1]))" "$0"; }
-CONTENTS="$(dirname "$(dirname "$(dirname "$(dirname "$(realpath "$0")")")")")"
+function realpath() { python -c "import os,sys; print(os.path.realpath(os.path.join(sys.argv[1],'..','..','..','..')))" "$0"; }
```

Comment:
At this point you could drop declaring `realpath` & go straight for `CONTENTS=$(python -c … )`

Reply:
agree, removed.